src/features/apiDelegation/components/OverviewPageContent/OrgDelegationActionBar/OrgDelegationActionBar.tsx

```diff
@@ -60,7 +60,7 @@ export const OrgDelegationActionBar = ({
             color={'second'}
             size={'medium'}
             onClick={softRestoreAllCallback}
-            aria-label={String(t('common.undo')) + ' ' + organization.orgName}
+            aria-label={String(t('common.undo')) + ' ' + organization.name}
```

Comment:
Should this include a reference to the action you are undoing? Something like: "undo delete orgName"?  

for example: aria-label={`{t('common.undo')} {t('api_delegation.delete')} {organization.name}`}

Reply:
And also, should we inform the user that the item will be deleted only after a confirm has been clicked?

Reply:
Not important(!) and not really related to this PR, so it might be that this is a task that needs its own issue in the backlog? IDK... 🤷🏼

Reply:
> Should this include a reference to the action you are undoing? Something like: "undo delete orgName"? 
> for example: aria-label={{t('common.undo')} {t('api_delegation.delete')} {organization.name}}
Great idea! I'll put it in right now (don't really see any reason to wait)

> And also, should we inform the user that the item will be deleted only after a confirm has been clicked?
Also a good idea, but I'm not sure exactly what is the best way to do so.
Just add a line that straight up tells them there's a save button they need to click on? Or is there a more elegant way to do it?
What do you think?

Reply:
Maybe we could change the text to something more descriptive like "add/remove {orgname} to delete list" or something like it?

Reply:
Yeah, but that's very programmer-specific language. Maybe "mark <the thing> for deletion"?
Or we could maybe just move the user from the button to the save button if you click on it to make it clear that you have to save? Or find some other way to inform the user that you need to save for this decision to come into effect?

I mean, that's what we do for the seeing users - inform them of the need to save by making a save button pop up and thus making them infer that they have to press it.

Reply:
I definitely agree "mark for deletion" is a more intuitive term.

I'm not so sure about moving the focus to the save button. Users may lose track of where they were in the interface, and if the user needs to make multiple changes before saving, having the focus repeatedly jump to the Save button can become annoying and inefficient.

Maybe we could use a live region to announce changes in the page content dynamically. After the user interacts with the delete button, the live region could announce "Changes made. Click the Save button to apply."

Reply:
Yeah, I think the live region might be the best solution.
Then we can announce that "<the thing> is set to be deleted, click the save button to save this change"

Reply:
Or yeah, "Changes made. Click the Save button to apply" is even easier!

Reply:
Done! 😊

Reply:
Nice! 👍🏼